### ISSUE TITLE
[next] get image optim loader working

### ISSUE THREAD

**Issue by jescalan** [static-dev/spike-core#217]
At the moment, webpack's image optimization plugin isn't working in spike@next. This is because the source isn't exposed in the same way when spike tries to extract and write it to a file. Just needs a bit more investigation!

**Comment by alexpate** [comment]
Is there any update on how to get this working?

**Comment by jescalan** [comment]
Not yet! I'd be happy to work with you to try to fix it if you want. A very easy fix is just running your image folder through imageoptim one time though.

**Comment by alexpate** [comment]
Sure thing! Is this something that has worked with previous versions of Spike?

**Comment by jescalan** [comment]
Yep, it worked with webpack1

**Comment by mazli84** [comment]
Any news on this issue? I still cannot get it working...

**Comment by jescalan** [comment]
Nope, it just hasn't been a priority for me. You can just run your images through imageoptim once and it's solved. You are very welcome to look into contributing a fix though if you want!
